maintainers/maintainer-list.nix

```diff
@@ -12992,6 +12992,12 @@
     githubId = 152312;
     name = "Periklis Tsirakidis";
   };
+  perstark = {
+    email = "perstark.se@gmail.com";
+    github = "perstarkse";
+    githubId = 63069986;
+    name = "Per Stark";
+  };
```

Comment:
Please move this change to a separate commit titled "maintainers: add perstark", preceeding the changes in blinkstick.